pkg/names/names.go

```diff
@@ -1,7 +1,36 @@
 package names
 
+import (
+	"os"
+)
+
 // ManifestDir is the directory where manifests are located.
 var ManifestDir = "./bindata"
 
 // NMStateResourceName is the name of the CR that the operator will reconcile
 const NMStateResourceName = "nmstate"
+
+// Relationship labels
+const COMPONENT_LABEL_KEY = "app.kubernetes.io/component"
+const PART_OF_LABEL_KEY = "app.kubernetes.io/part-of"
+const VERSION_LABEL_KEY = "app.kubernetes.io/version"
+const MANAGED_BY_LABEL_KEY = "app.kubernetes.io/managed-by"
+
+func GetRelationshipLabels() map[string]string {
+	labels := map[string]string{}
+	mapLabelKeys := map[string]string{
+		"COMPONENT":  COMPONENT_LABEL_KEY,
+		"PART_OF":    PART_OF_LABEL_KEY,
+		"VERSION":    VERSION_LABEL_KEY,
+		"MANAGED_BY": MANAGED_BY_LABEL_KEY,
+	}
+
+	for key, label := range mapLabelKeys {
+		envVar := os.Getenv(key)
+		if envVar != "" {
+			labels[label] = envVar
+		}
+	}
+
+	return labels
+}
```

Comment:
Doing this everything is not needed since env vars are inmutable on pods, let's just labels the map at startup and use it in the code.

Reply:
is it fine to do this with a function closure ?
this way it will init it just the first time the function run and on the sequential runs it will use the cached values